### ISSUE TITLE
UI

### ISSUE THREAD

**Issue by TheAdventurer64** [organicmaps/organicmaps#2778]
After the new exit info update, I noticed that the UI looked weird:![image](https://user-images.githubusercontent.com/50059322/174495030-46d8adb0-ad6f-4a03-9e15-474188efef47.png)

From a consumer perspective, this looks off; Exit info usually doesn't have symbols like [ and < embedded in the text.

**Comment by biodranik** [comment]
@AntonM030481

**Comment by pm4rcin** [comment]
I've had same problems that shield roads were display similarly with [] and some other symbols but no icons. I think it has regressed with may update but I'm not sure.

**Comment by AntonM030481** [comment]
Current implementation with comparison with Waze explained [here](https://github.com/organicmaps/organicmaps/issues/849#issuecomment-1163241894).

**Comment by TheAdventurer64** [comment]
@AntonM030481 would it be possible to design a more user friendly look for the destination info, even with the lack of road shield rendering?

**Comment by AntonM030481** [comment]
Currently it was implemented in [Waze style](https://github.com/organicmaps/organicmaps/issues/849#issuecomment-1161814283).
What are your suggestions?

**Comment by pm4rcin** [comment]
I think that Google maps has it implemented very well because it's verbose so you can read what you need to do. IMO road shields and exit numbers are very important especially on highways and other fast roads so you can look at which road number you have to drive and follow the physical road signs easier. Street names are more of a nice addition but they could be helpful in some cases.

**Comment by TheAdventurer64** [comment]
> Currently it was implemented in [Waze style](https://github.com/organicmaps/organicmaps/issues/849#issuecomment-1161814283).
> 
> What are your suggestions?

The way it looks now it looks weird and unfinished. Usually, exit info doesn't have things like brackets and inequality signs. 

TD:LR, I think that the destination info needs to be formatted to look more "polished".

**Comment by TheAdventurer64** [comment]
What are the steps needed to implement this in Organic Maps?

**Comment by biodranik** [comment]
@AntonM030481 ?

**Comment by AntonM030481** [comment]
I prefer to keep ">" and not change to ":" before destination.
":" is reserved to use after exit numbers (in OM the same as in Waze).

" / " for separation of destinations - are you sure that it's much better? "; " takes less space.

**Comment by TheDudeCode** [comment]
The current implementation looks like an equation or regex. I can see that it doesn't bother developers, but for any other person it looks very weird

**Comment by zyphlar** [comment]
I think the ultimate fix is to render basic "shields" (colored text boxes) as an intermediary step to fully localized realistic shields. I agree that something like `Exit 23: [US 45] 67th Street; City Center` is too much undifferentiated text for such a small space, by the time your eyes finish parsing it all you've rear ended someone (and in America, the exit number is often the most important, yet sometimes even that is unavailable like at interchanges and it's absolutely critical to match up "City Center" with a lane or add 30 minutes to your trip.) Your eyes becoming accustomed to looking at a yellow or green or blue square at a predictable part of the screen can literally be life or death here.

My PR #3118 adding streets to voice notifications can help with this: ideally the user doesn't even have to look at the screen to make the right turn.

**Comment by TheAdventurer64** [comment]
> I think the ultimate fix is to render basic "shields" (colored text boxes) as an intermediary step to fully localized realistic shields. I agree that something like `Exit 23: [US 45] 67th Street; City Center` is too much undifferentiated text for such a small space, by the time your eyes finish parsing it all you've rear ended someone (and in America, the exit number is often the most important, yet sometimes even that is unavailable like at interchanges and it's absolutely critical to match up "City Center" with a lane or add 30 minutes to your trip.) Your eyes becoming accustomed to looking at a yellow or green or blue square at a predictable part of the screen can literally be life or death here.
> 
> My PR #3118 adding streets to voice notifications can help with this: ideally the user doesn't even have to look at the screen to make the right turn.

Agreed. If I remember correctly, that was one of the todos listed on the original pull request that added destinations to the UI.
